### PR TITLE
pkglistgen: remove default-support-status option

### PR DESCRIPTION
**Pull request by coolo**
The OUTPUT groups need to specify the status now or it will default to
unsupported (which should still be fine for leap/factory)

### CODE REVIEW

File: pkglistgen.py [hunk 1]
```diff
@@ -1135,6 +1135,12 @@ def do_update_and_solve(self, subcmd, opts):
 
         if not opts.project:
             raise ValueError('project is required')
+        opts.staging_project = None
+        if opts.scope.startswith('staging:'):
```
Comment (lnussel) [inline pkglistgen.py]: how about not doing this here but instead use a few lines further down
if opts.scope.startswith("staging"), then try the match and set letters accordingly?

File: pkglistgen.py [hunk 2]
```diff
@@ -1177,6 +1183,8 @@ def do_update_and_solve(self, subcmd, opts):
         elif opts.scope == 'staging':
```
Comment (lnussel) [inline pkglistgen.py]:
```
  if opts.scope.startswith('staging'):
    if ':' in opts.scope:
       letters = opts.scope.split(':')[1]
    else:
       letters = api.get_staging_projects_short()
```

File: pkglistgen.py (hunk 1 above)
Comment (coolo) [reply pkglistgen.py]: the problem is the line in between those you commented - staging:X is not in SCOPES

**Comment by coolo** [comment]
The feedback of Ludwig won't work and I'm running this for 16 days already off branch

**Pull request merged**